playground-next13-live-editing/components/StoryblokProvider.tsx

```diff
@@ -0,0 +1,21 @@
+/** 1. Tag it as client component */
+"use client";
+import { storyblokInit, apiPlugin } from "@storyblok/react/rsc";
+
+/** 2. Import your components */
+import Page from "./Page";
+import Teaser from "./Teaser";
+
+/** 3. Initialize it as usual */
+storyblokInit({
```

Comment:
you also call `storyblokInit` also in layout.tsx. Maybe you can try to use only one ?

Reply:
Hey @jpkha we're doing that on purpose. The idea is not initialize the connection with Storyblok on both server and client side